src/librustdoc/html/render/print_item.rs

```diff
@@ -1745,7 +1745,13 @@ fn item_variants(
 }
 
 fn item_macro(w: &mut Buffer, cx: &mut Context<'_>, it: &clean::Item, t: &clean::Macro) {
-    highlight::render_item_decl_with_highlighting(&t.source, w);
+    wrap_item(w, |w| {
+        // FIXME: Also print `#[doc(hidden)]` for `macro_rules!` if it `is_doc_hidden`.
+        if !t.macro_rules {
+            write!(w, "{}", visibility_print_with_space(it, cx));
+        }
+        highlight::write_code(w, &t.source, None, None);
```

Comment:
I do think we should just stop highlighting macros.

1. Here's the original PR that added it: https://github.com/rust-lang/rust/pull/12647

2. I notice that, in the original implementation, the highlighter accomplished more because the *body source code* was shown.

3. A later pull request made it only show the arm, making the highlighter less useful, but not removing it, and no reason for keeping the highlighter was given AFAICT: https://github.com/rust-lang/rust/pull/30074

Reply:
Applied.